### ISSUE TITLE
Internal Server Error if song does not exist and simple is true

### ISSUE THREAD

**Issue by PolyAnthi** [Dhravya/dhravyaAPI#35]
![image](https://user-images.githubusercontent.com/79588299/148650044-24f256df-7ff4-4398-9ac3-44628cfe571f.png)

If ?simple=true and you input a song which does not exist, instead of responding with success 0/404 code, it returns server error

**Comment by itsnotrin** [comment]
Can this issue be deleted since we're removing simple? @Dhravya

**Comment by Dhravya** [comment]
Yes you can delete it Sent from Mail for Windows From: itsnotrinSent: 27 January 2022 23:45To: Dhravya/dhravyaAPICc: Dhravya Shah; MentionSubject: Re: [Dhravya/dhravyaAPI] Internal Server Error if song does not exist and simple is true (Issue #35) Can this issue be deleted since we're removing simple? @Dhravya—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>
